src/engine/openrpc/schemas/payload.yaml

```diff
@@ -0,0 +1,202 @@
+PayloadStatusV1:
+  title: Payload status object V1
+  type: object
+  required:
+    - status
+  properties:
+    status:
+      title: Payload validation status
+      type: string
+      enum:
+        - VALID
+        - INVALID
+        - SYNCING
+        - ACCEPTED
+        - INVALID_BLOCK_HASH
+    latestValidHash:
+      title: The hash of the most recent valid block
+      $ref: '#/components/schemas/hash32'
+    validationError:
+      title: Validation error message
+      type: string
+RestrictedPayloadStatusV1:
+  $ref: '#/components/schemas/PayloadStatusV1'
+  properties:
+    status:
+      $ref: '#/components/schemas/PayloadStatusV1/properties/status'
+      description: Set of possible values is restricted to VALID, INVALID, SYNCING
+      enum:
+        - VALID
+        - INVALID
+        - SYNCING        
+    latestValidHash:
+      $ref: '#/components/schemas/PayloadStatusV1/properties/latestValidHash'
+    validationError:
+      $ref: '#/components/schemas/PayloadStatusV1/properties/validationError'
+PayloadStatusV2:
```

Comment:
I don't see this defined in any spec?

Reply:
Okay I see. I think it would be better to either define it explicitly in the markdown with `V2` or to use this format `PayloadStatusNoInvalidBlockHash`. There might be a collision at some point when we do need a `V2` of `PayloadStatus`.

Reply:
I would not change the markdown as we're too far in the Shanghai release pipeline now, so, I am leaning to the second option